django/contrib/gis/geos/collections.py

```diff
@@ -134,8 +136,11 @@ class MultiPolygon(GeometryCollection):
 
     @property
     def cascaded_union(self):
-        "Returns a cascaded union of this MultiPolygon."
-        return GEOSGeometry(capi.geos_cascaded_union(self.ptr), self.srid)
```

Comment:
After some thinking about difference between guaranteed and actual behavior in general I decided to leave the old implementation.